ebusd-2.1.x/en/vaillant/08.hmu00.HW5103.SW0901.csv

```diff
@@ -0,0 +1,210 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment,field2,part (m/s),datatypes/templates,divider/values,unit,comment,field3,part (m/s),datatypes/templates,divider/values,unit,comment,field4,part (m/s),datatypes/templates,divider/values,unit,comment
+
+r,hmu,Standby,,,,B503,020003,,,HEX:6,,,,,,,,,,,,,,,,,,,,,
+# START Unknown messages,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+r,hmu,unknownMPoll60sB507h,,,08,B507,0900,,,HEX:*,,,Unknown poll every 60 seconds from CTLV2 to HMU,,,,,,,,,,,,,,,,,,
+#r,hmu,HeatpumpLoad,,,08,B507,0931,,,percent1,,,Heatpump Load (percent),,,,,,,,,,,,,,,,,,
+#r,hmu,unknownMPoll60sB511h,,,08,B511,00,,,HEX:*,,,Unknown poll every 60 seconds from CTLV2 to HMU,,,,,,,,,,,,,,,,,,
+
+*r,,,,,,B51A,,,,IGN:1,,,,,,,,,,,,,,,,,,,,,
+r,,unknownPoll60sB51Ah2,,,08,,32,,,HEX:*,,,Unknown poll every 60 seconds from VWZ to HMU,,,,,,,,,,,,,,,,,,
+r,,unknownPoll60sB51Ah3,,,08,,33,,,HEX:*,,,Unknown poll every 60 seconds from VWZ to HMU,,,,,,,,,,,,,,,,,,
+r,,unknownPoll60sB51Ah4,,,08,,34,,,HEX:*,,,Unknown poll every 60 seconds from VWZ to HMU,,,,,,,,,,,,,,,,,,
+r,,unknownPoll60sB51Ah5,,,08,,35,,,HEX:*,,,Unknown poll every 60 seconds from VWZ to HMU,,,,,,,,,,,,,,,,,,
+r,,unknownPoll60sB51Ah6,,,08,,36,,,HEX:*,,,Unknown poll every 60 seconds from VWZ to HMU,,,,,,,,,,,,,,,,,,
+
+*r,,,,,,B51A,04,,,,,,,,,,,,,,,,,,,,,,,,
+r,,unknownB51Ah2634,,,08,,2634,,,HEX:*,,,Unknown from VWZ to HMU,,,,,,,,,,,,,,,,,,
+r,,unknownB51Ah0532,,,08,,0532,,,HEX:*,,,Unknown from VWZ to HMU,,,,,,,,,,,,,,,,,,
+r,,unknownB513h0528,,,08,B513,0528,,,HEX:*,,,Unknown from CTLV2 to HMU,,,,,,,,,,,,,,,,,,
+r,,unknownB509h2802,,,08,B509,2802,,,HEX:*,,,Unknown from VWZ to HMU,,,,,,,,,,,,,,,,,,
+# END Unknown messages,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+
+r,,Standby,,,,B503,020003,,,HEX:6,,,,,,,,,,,,,,,,,,,,,
+r,,HeatpumpLoadSensor,,,08,B507,0931,,,percent1,,,Heatpump Load Sensor (percent),,,,,,,,,,,,,,,,,,
+
+# Values from VWZ AI Plus Test Menu,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+#(queried from WP entries Installer level/Test Menu/Sensor test),,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+*r,,,,,,B514,05,,,IGN:2,,,,,,,,,,,,,,,,,,,,,
+# removed all messages that yielded ffff,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+
+r,,BuildingCircuitPumpPower,,,,,0103FFFF,,,percent3,,percent,Building circuit pump power (percent of maximum),,,,,,,,,,,,,,,,,,
```

Comment:
vaillant/08.hmu00.HW5103.SW0901.csv:31: ERR: element not found, field type PERCENT3 in field 1